pkg/config/config.go

```diff
@@ -167,6 +168,21 @@ func (c *Config) Validate() []error {
 			}
 		}
 
+		permittedLicenses := []string{"GSuiteEnterprise", "GSuiteBusiness", "GSuiteBasic", "GSuiteEssentials", "GSuiteLite", "GoogleAppsMessageSecurity", "GSuiteEducation", "GSuiteEducationStudent", "GoogleDrive20GB", "GoogleDrive50GB", "GoogleDrive200GB", "GoogleDrive400GB", "GoogleDrive1TB", "GoogleDrive2TB", "GoogleDrive4TB", "GoogleDrive8TB", "GoogleDrive16TB", "GoogleVault", "GoogleVaultFormerEmployee", "CloudIdentity", "CloudIdentityPremium", "GoogleVoiceStarter", "GoogleVoiceStandard", "GoogleVoicePremier"}
```

Comment:
You would have to keep this manually in-sync with glib/data.go's license list, right? If so, it might be a better idea to move the data into its own package (`pkg/glib/data` maybe?), and reference the data from the glib and the config packages.